components/integration/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+resources:
+- allow-argocd-to-manage.yaml
+- argocd-permissions.yaml
+- https://github.com/redhat-appstudio/integration-service/config/default?ref=f8e0590af265572ac725e720c06c527dcaa1592e
```

Comment:
```
$ kustomize build components/integration
Error: accumulating resources: accumulation err='accumulating resources from 'https://github.com/redhat-appstudio/integration-service/config/default?ref=f8e0590af265572ac725e720c06c527dcaa1592e': yaml: line 164: mapping values are not allowed in this context': recursed accumulation of path '/tmp/kustomize-552351906/config/default': accumulating resources: accumulation err='accumulating resources from '../crd': '/tmp/kustomize-552351906/config/crd' must resolve to a file': couldn't make target for path '/tmp/kustomize-552351906/config/crd': error converting YAML to JSON: yaml: line 9: could not find expected ':'
```

Reply:
Thanks, fixing it and will update this PR with the new sha

Reply:
Would it be possible to test first before asking for review?

```
Failed to pull image "controller:latest": rpc error: code = Unknown desc = reading manifest latest in docker.io/library/controller: errors: denied: requested access to the resource is denied unauthorized: authentication required 
```

Reply:
Nope, I've been instructed to leave all testing to a certain @Michkov  ;)
Of course it was tested and worked without running into that issue. Investigating.

Reply:
Just double checked and am finding no issue with using that image. At least two other controllers are also using it already. Will try to retest to eliminate any API limit issues or similar.

Reply:
`kustomize build components/integration/`
Deployment integration-service-controller-manager is using `image: controller:latest` so that should be replaced.